tests/tests/test_filetransfer.py

```diff
@@ -281,4 +323,406 @@ def test_filetransfer(self, enterprise_no_client):
 
         authtoken = auth.get_auth_token()
 
-        super().test_filetransfer(devices[0], authtoken)
+        super().test_filetransfer(devices[0], authtoken, content_assertion="ServerURL")
+
+
+class TestFileTransferLimits:
+    def test_filetransfer_limits_setup(self, standard_setup_one_client):
```

Comment:
You shouldn't call this function "test_", otherwise it will also be executed as an independent test, which is not what you want. Maybe `setup_filetransfer_limits`?

Reply:
right! (it also did not hurt to execute it as an extra test) fixed, thanks.